### ISSUE TITLE
Issue connecting to the profile database

### ISSUE THREAD

**Issue by JavideJ** [rconcep/snl-quest#6]
I have installed QuESt and run it satisfactorily but I fail when trying to download (for example) an utility rate structure. The next message appears: "There was an issue connecting to and downloading the lists of utilities. Check your connection settings and try again". This is the screenshot of what´s happening. I have my own API key.

Thanks

![Captura](https://user-images.githubusercontent.com/50321196/57217676-8fb12c80-6ff3-11e9-9216-ad655027ee03.PNG)

**Comment by rconcep** [comment]
Hello,
Could you grab the console/terminal output when encountering this error? Also, could you see if you have the same error when trying to download other data in QuESt Data Manager (e.g., load profile data)?

You may need to configure proxy/SSL verification settings depending on your internet connection. For example, on my corporate network I need to set my proxy settings in QuESt if they are not configured in my Windows environment variables.

**Comment by JavideJ** [comment]
Yes, it also happens when trying to download other data. This is what is displayed in the console:

[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.
[WARNING] [RateStructureDM] Hit download retry limit.
[ERROR  ] [RateStructureDM] Failed to establish a connection to the host server.

Thanks

**Comment by rconcep** [comment]
OK, based on your first screenshot, the exception based on a simple request (before the API key matters). Here is the URL it is trying to grab:

https://openei.org/doe-opendata/dataset/53490bd4-671d-416d-aae2-de844d2d2738/resource/500990ae-ada2-4791-9206-01dc68e36f12/download/iouzipcodes2017.csv

I'm assuming if you put that into a browser it'll work fine. 

The exception you are getting is from a `requests.ConnectionError`:

> In the event of a network problem (e.g. DNS failure, refused connection, etc), Requests will raise a ConnectionError exception.

You can try this code snippet in Python which is what is throwing the exception:

```
import requests

with requests.Session() as req:
    URL_OPENEI_IOU = https://openei.org/doe-opendata/dataset/53490bd4-671d-416d-aae2-de844d2d2738/resource/500990ae-ada2-4791-9206-01dc68e36f12/download/iouzipcodes2017.csv
    http_request = req.get(URL_OPENEI_IOU,)
```

You'll probably get the exception and probably more detail about what is causing it. I suspect that you will need to supply some arguments to `req.get()` to get the request to work.
